### PR TITLE
Document architecture

### PR DESCRIPTION
**Pull request by flavio**
This resumes the work initiated by @ereslibre with PR https://github.com/kubewarden/docs/pull/24

This addresses https://github.com/kubewarden/docs/issues/13

### CODE REVIEW

**Review by ereslibre — CHANGES_REQUESTED** [review]
LGTM, a couple of comments.

File: src/architecture.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+naming convention: `/validate/<policy ID>`.
+
+This is how the cluster looks like once the initialization of `policy-server`
+is completed:
+
+![policy-server initialized](/images/architecture_sequence_02.jpg)
+
+### Making Kubernetes aware of the policy
+
+The `policy-server` Pods have a
```
[the remark this answers is not in the document]
Comment (flavio) [reply src/architecture.md:112]: it's `Pods`, plural not `Pod` ;)

**Comment by flavio** [comment]
I accepted the change and squashed it into my commit. Merging

File: src/architecture.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ereslibre) [reply src/architecture.md:112]: My brain skipped Pods completely... ugh

**Pull request merged**